CentralComputing/NetworkManager.hpp

```diff
@@ -45,172 +46,66 @@ struct Network_Command {
 };
 
 
-static int socketfd;
-static int clientfd;
-static int udp_socket;
-
-static std::atomic<bool> running(false);
-static sockaddr_storage addr_dest = {};
-static SafeQueue<Network_Command *> network_queue;
+extern int socketfd;
```

Comment:
It makes sense now why this needs to be `extern`. 
https://stackoverflow.com/questions/11623451/static-vs-non-static-variables-in-namespace
Namespaces are hard